docs/DeveloperGuide.md

```diff
@@ -154,6 +154,59 @@ Classes used by multiple components are in the `seedu.ccacommander.commons` pack
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Enrol Feature
+The enrol feature allows users to link a member to an event in order to keep track of their attendance and contributions to the event.
+
+This section will explain how the enrol feature was implemented and the various design considerations when implementing the feature.
+
+#### Implementation
+The enrollment mechanism is facilitated by `EnrolCommand`. It extends `Command`.
```

Comment:
I think the correct spelling here is "enrolment"?

Reply:
Fixed!